magni_bringup/README.md

```diff
@@ -64,4 +64,4 @@ If new additions need to be added to core.launch, add them into `magni_bringup/p
    - b) extrinsics in such format can easily be modified by non-expert users (no touching the code or URDF files)
    - c) extrinsics in such format can easily be modified programmatically eg. by auto-extrinsics-calibration programs
 
- - We decided to move many motor controller and other key operating mode parameters from base.yaml to robot.yaml. A lot of work was done to do this by Rohan but the issue has been only creaping along and needs closure so that it will finally be safe to use magni_robot repository along with ubiquity_motor and our main codeline will then be more supportable again.
\ No newline at end of file
+ - We decided to move many motor controller and other key operating mode parameters from base.yaml to robot.yaml. A lot of work was done to do this by Rohan but the issue has been only creeping along and needs closure so that it will finally be safe to use magni_robot repository along with ubiquity_motor and our main codeline will then be more supportable again.
```

Comment:
This may likely be the MOST important fix of the entire change!   ;-)

Reply:
Very important indeed :D